README.md

```diff
@@ -83,16 +83,19 @@ See the [SMART Launch](./examples/smart-launch.js) example with OAuth2 support.
 # Logging
 
 The [debug library](https://www.npmjs.com/package/debug) can provide logging
-during development. Two different logging namespaces are provided, `node-fhir-
-client:info` logs each request and response, and `node-fhir-client:error` logs
+during development. Two different logging namespaces are provided, `fhir-kit-
+client:info` logs each request and response, and `fhir-kit-client:error` logs
 errors. To enable logging during development, add one of the namespaces to the
-DEBUG environment variable, or use `node-fhir-client:*` to enable both.
+DEBUG environment variable, or use `fhir-kit-client:*` to enable both.
+
 ```
-$ DEBUG=node-fhir-client:* node examples/index.js
+$ DEBUG=fhir-kit-client:* node smart-launch.js
 ```
 
 # Todo
 
+
+
 # Contributing
 
 All are welcome to contribute. By participating in this project you agree to
```

Comment:
The url for `Code of Conduct` link in the next line of the Readme needs updating as well.